src/js/modules/analysis.js

```diff
@@ -1281,24 +1281,27 @@ const DownloadTabView = Backbone.View.extend({
     render() {
         const that = this;
         // Order
-        const groupsKeys = [
+        const order = [
             'Sequence data',
             'Functional analysis',
             'Pathways and Systems',
             'Taxonomic analysis',
             'Taxonomic analysis SSU rRNA',
             'Taxonomic analysis LSU rRNA',
             'Taxonomic analysis ITS',
+            'Taxonomic analysis mOTU',
             'non-coding RNAs'
         ];
         const data = that.model.attributes.downloadGroups;
+        const dataKeysOrdered = _.sortBy(_.allKeys(data), (dataKey) => {
```

Comment:
That is a good working approach. At least this way we won't loose any new entries in the future. Also we might want to consider to store the order of each item in the database in the future.

Reply:
Agree!